ui-v2/app/components/hashicorp-consul/pageobject.js

```diff
@@ -47,5 +47,8 @@ export default (clickable, attribute, is, authForm) => scope => {
     ':checked',
     '[data-test-nspace-menu] > input[type="checkbox"]'
   );
+  page.navigation.dcs = collection('[data-test-datacenter-picker]', {
+    name: clickable('a'),
+  });
```

Comment:
We moved the `dcs` in the pageobject to be part of the navigation rather than the page (previously they were attached only to the service page). This means both the current DC and the available DCs are available on all pages.